src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PETypeParameterSymbol.cs

```diff
@@ -276,6 +276,17 @@ public override bool HasReferenceTypeConstraint
             }
         }
 
+        internal override bool HasNullableReferenceTypeConstraint
+        {
+            get
+            {
+                // PROTOTYPE(NullableReferenceTypes): Support external annotations.
+                return HasReferenceTypeConstraint && 
+                       ((PEModuleSymbol)this.ContainingModule).Module.HasNullableAttribute(_handle, out ImmutableArray<bool> nullableTransformFlags) &&
```

Comment:
> Could you update the markdown document to describe this metadata representation? It will be useful for F#.

Will do. What markdown document in particular?

---
In reply to: [211789391](https://github.com/dotnet/roslyn/pull/29407#discussion_r211789391) [](ancestors = 211789391)